### PR TITLE
Clean up ignore files

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -62,8 +62,14 @@ rules:
...
 
 exclude:
+  - app.coffee
+  - CONTRIBUTING.md
   - CNAME
-  - Gemfile*
+  - Gemfile
+  - Gemfile.lock
+  - LICENSE.md
+  - licenses.json
```
Comment (tekkub) [inline _config.yml]: `licenses.json` is actually intended to be generated and served up as a sort of API.  It shouldn't be excluded.

Comment (XhmikosR) [inline _config.yml]: Is it used atm? Because I see it isn't.

Comment (tekkub) [inline _config.yml]: Yup: http://choosealicense.com/licenses.json

Comment (XhmikosR) [inline _config.yml]: OK. How about CNAME? Why is that excluded?

Comment (tekkub) [inline _config.yml]: CNAME is used by github's server to set things up, it doesn't need to be served to users.

Comment (XhmikosR) [inline _config.yml]: OK. So this patch is ok I guess; I updated the PR earlier.

**Pull request merged**